pages/cart.vue

```diff
@@ -134,38 +137,64 @@ export default defineComponent({
     })
 
     const cartItems = computed(() => cartGetters.getItems(cart.value))
-    const cartOrder = computed(() => cartGetters.getTotals(cart.value))
+    const cartOrder = computed(() => {
```

Comment:
`cartOrder` is a confusing name and re assigning 'total' here to be discountedSubtotal or total is confusing.

I think this would be better if we just made computed props for all of these values.